codegen/parser.py

```diff
@@ -337,7 +337,8 @@ def special_case_time_fields(
                     raise NotImplementedError(
                         f"Unknown timedelta primitive type: {values['type']!r}"
                     )
-            return values | {
```

Comment:
mypy started choking with `no-any-return` here, changing to non-union operator initialization fixed it.